event-driven-architecture-topologies.md

```diff
@@ -4,7 +4,7 @@ The event-driven architecture is a style of architecture that is used to develop
 
 Most applications use what is called request-based model, where the client sends a request to the specific endpoint of the server and waits for the response.
 
- In event-based model, applications takes actions when something happens in the system. One of the applications publishes an event, and other applications subscribe to that event. When the event is published, all the subscribers are notified and they can take actions based on the event. This is a very powerful model that allows us to build highly scalable and loosely coupled applications.
+ In event-based model, applications take action when something happens in the system. One of the applications publishes an event, and other applications subscribe to that event. When the event is published, all the subscribers are notified and they take actions based on the event. The event-based model is a very powerful model that allows us to build highly scalable and loosely coupled applications.
```

Comment:
The event-based model, on the other hand, puts events at the center. This model allows [Loose coupling]((https://en.wikipedia.org/wiki/Loose_coupling) between the components of a system.